app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -224,12 +227,30 @@
     }
   }
 
+  @include govuk-media-query($until: tablet) {
+    .gem-c-feedback__option-list-item--useful,
+    .gem-c-feedback__option-list-item--not-useful,
+    .gem-c-feedback__option-list-item--wrong {
+
+      position: relative;
+
+      a:after {
```

Comment:
We don't style using selectors like this. I think this should be on the `.gem-c-feedback__prompt-link` class?

Reply:
have updated